package on2015_04.on2015_04_02_Codeforces_Round__143__Div__2_.D___Magic_Box;

import net.egork.chelper.tester.NewTester;

import org.junit.Assert;
import org.junit.Test;

public class Main {
	@Test
	public void test() throws Exception {
		if (!NewTester.test("lib/test/on2015_04/on2015_04_02_Codeforces_Round__143__Div__2_/D___Magic_Box/D - Magic Box.task"))
			Assert.fail();
	}
}
